server/apolloServer.js

```diff
@@ -0,0 +1,42 @@
+const { ApolloServer } = require('apollo-server-express')
+
+function newApolloServer (app, schema, httpServer, tracing, playgroundConfig, graphqlEndpoint, securityService) {
+  let AuthContextProvider = null
+
+  if (securityService) {
+    AuthContextProvider = securityService.getAuthContextProvider()
+  }
+
+  let apolloServer = new ApolloServer({
+    schema,
+    context: async ({ req }) => {
+      const context = {
+        request: req
+      }
+      if (AuthContextProvider) {
+        context.auth = new AuthContextProvider(req)
+      }
+      return context
+    },
+    tracing,
+    playground: {
```

Comment:
Now that I think of it, all this stuff could probably be refactored into `server/config/index.js`

Reply:
LOL. This is what I have suggested, but forgot to click send on friday :)